jetstream.go

```diff
@@ -252,9 +252,19 @@ func (m *Msg) AckProgress(opts ...AckOption) error {
 	return m.jsAck(AckProgress, opts...)
 }
 
-// AckNext performs an Ack() and request that the next message be sent to subject ib
-func (m *Msg) AckNext(ib string) error {
-	return m.RespondMsg(&Msg{Subject: m.Reply, Reply: ib, Data: AckNext})
+// AckNextMulti performs an Ack() and request that the next count message be sent to subject inbox
+func (m *Msg) AckNextMulti(inbox string, count uint) error {
```

Comment:
@derekcollison did this before realising we also want to support the full request structure, so what API do you suggest? `AckNextMulti(inbox, count, timeout)`? where timeout is a duration.

Reply:
Need to think about this one a bit more.. Will try to do so a bit later today.

Reply:
gentle reminder about this one :)

Reply:
Since neither of these will return a message or series of messages, meaning they just communicate where to deliver the next batch, we just need default and full blow request.

`AckNext(*nextReq)` where if nextReq is nil then assume same delivery subject and batch of 1.
Otherwise they can fill in a formal request?

Or we do this.

`AckNext(batchSize int)` and `AckNextRequest(nextReq)`

Reply:
we really want them to create that structure? Not do it for them by just asking the 2 possible values? Seems a lot easier this way than expect people to make a whole structure to me.

Reply:
We have more than 2 values and in this case the reply subject is probably same as from the message we are using to do AckNext. So really only batchSize is a common use case but could also set "don't wait flag" etc. 

If you think the request is fixed and we will not add any fields in we could try to explode it out instead.

Reply:
updated